content/blog/2020-09-21-introducing-the-new-jsx-transform.md

```diff
@@ -0,0 +1,245 @@
+Create React App support [has been added](https://github.com/facebook/create-react-app/pull/9645) and will available in the [upcoming v4.0 release](https://gist.github.com/iansu/282dbe3d722bd7231fa3224c0f403fa1) which is currently in beta testing.
+
+### Next.js
+
+Next.js [v9.5.3](https://github.com/vercel/next.js/releases/tag/v9.5.3)+ uses the new transform for compatible React versions.
+
+### Gatsby
+
+Gatsby [v2.24.5](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/CHANGELOG.md#22452-2020-08-28)+ uses the new transform for compatible React versions.
+
+### Manual Babel Setup
+
+Support for the new JSX transform is available in Babel [v7.9.0](https://babeljs.io/blog/2020/03/16/7.9.0) and above.
+
+First, you'll need to update to the latest Babel and plugin transform.
+
+If you are using `@babel/plugin-transform-react-jsx`:
+
+```bash
+# for npm users
+npm update @babel/core @babel/plugin-transform-react-jsx
+```
+
+```bash
+# for yarn users
+yarn upgrade @babel/core @babel/plugin-transform-react-jsx
+```
+
+If you are using `@babel/preset-react`:
+
+```bash
+# for npm users
+npm update @babel/core @babel/preset-react
+```
+
+```bash
+# for yarn users
+yarn upgrade @babel/core @babel/preset-react
+```
+
+Currently, the old transform (`"runtime": "classic"`) is the default option. To enable the new transform, you can pass `{"runtime": "automatic"}` as an option to `@babel/plugin-transform-react-jsx` or `@babel/preset-react`:
+
+```js
+// If you are using @babel/preset-react
+{
+  "presets": [
+    ["@babel/preset-react", {
+      "runtime": "automatic"
+    }]
+  ]
+}
+```
+
+```js
+// If you're using @babel/plugin-transform-react-jsx
+{
+  "plugins": [
+    ["@babel/plugin-transform-react-jsx", {
+      "runtime": "automatic"
+    }]
+  ]
+}
+```
+
+Starting from Babel 8, `"automatic"` will be the default runtime for both plugins. For more information, check out the Babel documentation for [@babel/plugin-transform-react-jsx](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx) and [@babel/preset-react](https://babeljs.io/docs/en/babel-preset-react).
+
+> Note
+>
+> If you use JSX with a library other than React, you can use [the `importSource` option](https://babeljs.io/docs/en/babel-preset-react#importsource) to import from that library instead -- as long as it provides the necessary entry points. Alternatively, you can keep using the classic transform which will continue to be supported.
+
+### ESLint
+
+If you are using [eslint-plugin-react](https://github.com/yannickcr/eslint-plugin-react), the `react/jsx-uses-react` and `react/react-in-jsx-scope` rules are no longer necessary and can be turned off or removed.
+
+```json
+"rules": {
+     "react/jsx-uses-react": "off",
+     "react/react-in-jsx-scope": "off"
+}
+```
+
+### TypeScript
+
+TypeScript support for the JSX transform is currently in progress and will be available soon! You can track the progress [here](https://github.com/microsoft/TypeScript/issues/34547).
```

Comment:
It's part of the 4.1 beta now: https://devblogs.microsoft.com/typescript/announcing-typescript-4-1-beta/#jsx-factories

Reply:
Oh nice. Want to amend the post?